devicon.json

```
@@ -5838,7 +5838,7 @@
     {
         "name": "php",
         "altnames": [
-            "phphypertextpreprocessor"
+            "PHP Hypertext Preprocessor"
```

Adding in the original name of PHP. 
Source: https://en.m.wikipedia.org/wiki/PHP#:~:text=PHP%20originally%20stood%20for%20Personal,recursive%20initialism%20PHP%3A%20Hypertext%20Preprocessor.
```suggestion
            "PHP Hypertext Preprocessor",
            "Personal Home Page"
```